<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorkspaceWidget</class>
 <widget class="QWidget" name="WorkspaceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>345</width>
    <height>292</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WorkspaceWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="2">
    <widget class="WorkspaceTreeView" name="workspaceTreeView">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="indentation">
      <number>15</number>
     </property>
     <property name="uniformRowHeights">
      <bool>true</bool>
     </property>
     <property name="animated">
      <bool>false</bool>
     </property>
     <property name="headerHidden">
      <bool>true</bool>
     </property>
     <property name="expandsOnDoubleClick">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QToolButton" name="btnAddDirectory">
       <property name="text">
        <string>Add folder</string>
       </property>
       <property name="icon">
        <iconset resource="../../icons.qrc">
         <normaloff>:/icons/folder_add.png</normaloff>:/icons/folder_add.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonIconOnly</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
       <property name="arrowType">
        <enum>Qt::NoArrow</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnLoadFile">
       <property name="text">
        <string>Load selected file</string>
       </property>
       <property name="icon">
        <iconset resource="../../icons.qrc">
         <normaloff>:/icons/page_next.png</normaloff>:/icons/page_next.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnRemoveFile">
       <property name="text">
        <string>Remove selected</string>
       </property>
       <property name="icon">
        <iconset resource="../../icons.qrc">
         <normaloff>:/icons/page_remove.png</normaloff>:/icons/page_remove.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonIconOnly</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
       <property name="arrowType">
        <enum>Qt::NoArrow</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnRefresh">
       <property name="text">
        <string>Refresh view</string>
       </property>
       <property name="icon">
        <iconset resource="../../icons.qrc">
         <normaloff>:/icons/repeat.png</normaloff>:/icons/repeat.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonIconOnly</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
       <property name="arrowType">
        <enum>Qt::NoArrow</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>185</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>WorkspaceTreeView</class>
   <extends>QTreeView</extends>
   <header>widgets/workspace/WorkspaceTreeView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../icons.qrc"/>
 </resources>
 <connections/>
</ui>
